### PR TITLE
docs: translation for input in react dom

### LINKED ISSUE #423: `<input>`
Page: https://id.react.dev/reference/react-dom/components/input

Assignee: @abidra

### PR DESCRIPTION
**Pull request by abidra**
Closes #423

Translation to Bahasa for Input in React Dom.

Reference: #330

### CODE REVIEW

File: src/content/reference/react-dom/components/input.md [hunk 1]
```diff
@@ -456,140 +456,140 @@ function SignupForm() {
...
+Jika ini tidak menyelesaikan masalah, mungkin saja masukan dihapus dan ditambahkan kembali dari DOM pada setiap penekanan tombol. Ini dapat terjadi jika Anda secara tidak sengaja [menyetel ulang *state*](/learn/preserving-and-resetting-state) pada setiap render ulang, contohnya jika masukan atau salah satu parentnya selalu menerima atribut `key` yang berbeda, atau jika Anda menyusun definisi fungsi komponen (yang tidak didukung dan menyebabkan komponen "dalam" selalu dianggap pohon yang berbeda).
 
 ---
 
-### I'm getting an error: "A component is changing an uncontrolled input to be controlled" {/*im-getting-an-error-a-component-is-changing-an-uncontrolled-input-to-be-controlled*/}
+### Saya menerima pesan kesalahan: "Komponen sedang mengubah masukan yang tidak terkontrol menjadi dikontrol" {/*im-getting-an-error-a-component-is-changing-an-uncontrolled-input-to-be-controlled*/}
 
 
-If you provide a `value` to the component, it must remain a string throughout its lifetime.
+Jika Anda memberikan `value` ke komponen, komponen tersebut harus tetap berupa string selama masa pakainya.
```
Comment (eriansha) [inline src/content/reference/react-dom/components/input.md:589]: i think "it" in this context is referring to the `value`, not the component. In this case, "komponen" should be replaced with "nilai"

Comment (abidra) [reply src/content/reference/react-dom/components/input.md:589]: yes correct, thanks for the feedback @eriansha!

File: src/content/reference/react-dom/components/input.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "<input>"
 
 <Intro>
 
-The [built-in browser `<input>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) lets you render different kinds of form inputs.
+Komponen [bawaan browser `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) memungkinkan Anda me-*render* berbagai jenis masukan form.
```
Comment (FelixFern) [inline src/content/reference/react-dom/components/input.md:7]: "browser" should be written as "peramban" according to the https://github.com/reactjs/id.react.dev/wiki/Glosarium

File: src/content/reference/react-dom/components/input.md [hunk 3]
```diff
@@ -16,92 +16,92 @@ The [built-in browser `<input>` component](https://developer.mozilla.org/en-US/d
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `<input>` {/*input*/}
 
-To display an input, render the [built-in browser `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) component.
+Untuk menampilkan sebuah masukan, render komponen [bawaan browser `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
```
Comment (FelixFern) [inline src/content/reference/react-dom/components/input.md:23]: "browser" should be written as "peramban" according to the https://github.com/reactjs/id.react.dev/wiki/Glosarium

File: src/content/reference/react-dom/components/input.md [hunk 4]
```diff
@@ -146,11 +146,11 @@ input { margin: 5px; }
 
 ---
 
-### Providing a label for an input {/*providing-a-label-for-an-input*/}
+### Memberikan label untuk sebuah masukan {/*providing-a-label-for-an-input*/}
 
-Typically, you will place every `<input>` inside a [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) tag. This tells the browser that this label is associated with that input. When the user clicks the label, the browser will automatically focus the input. It's also essential for accessibility: a screen reader will announce the label caption when the user focuses the associated input.
+Biasanya, Anda akan menaruh setiap `<input>` di dalam sebuah tag [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). Ini memberi tahu browser bahwa label ini dikaitkan dengan masukan itu. Saat pengguna melakukan klik pada label, browser akan secara otomatis memfokuskan input. Ini juga penting untuk aksesibilitas: pembaca layar akan memberitahukan keterangan label saat pengguna memfokuskan masukan terkait.
```
Comment (FelixFern) [inline src/content/reference/react-dom/components/input.md:151]: "browser" should be written as "peramban" according to the https://github.com/reactjs/id.react.dev/wiki/Glosarium

File: src/content/reference/react-dom/components/input.md [hunk 5]
```diff
@@ -16,92 +16,92 @@ The [built-in browser `<input>` component](https://developer.mozilla.org/en-US/d
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `<input>` {/*input*/}
 
-To display an input, render the [built-in browser `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) component.
+Untuk menampilkan sebuah masukan, render komponen [bawaan peramban `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
```
Comment (r17x) [inline src/content/reference/react-dom/components/input.md:23]:
```suggestion
Untuk menampilkan sebuah masukan, *render* komponen [bawaan peramban `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
```

**Review by r17x — DISMISSED** [review]
All words like "render" need to be italic

**Comment by mhaidarhanif** [comment]
@abidra need to resolve the requests from @r17x

**Comment by abidra** [comment]
> render

done

**Comment by mhaidarhanif** [comment]
@r17x can approve so this can be merged soon

File: src/content/reference/react-dom/components/input.md [hunk 6]
```diff
@@ -16,92 +16,92 @@ The [built-in browser `<input>` component](https://developer.mozilla.org/en-US/d
...
+
+Props `<input>` ini relevan untuk masukan tidak terkontrol dan masukan terkontrol:
+
+* [`accept`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#accept): Sebuah string. Menentukan tipe file mana yang diterima oleh masukan `type="file"`.
+* [`alt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#alt): Sebuah string. Menentukan teks gambar alternatif untuk masuk `type="image"`.
+* [`capture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#capture): Sebuah string. Menentukan media (mikrofon, video, atau kamera) yang ditangkap oleh masukan `type="file"`.
+* [`autoComplete`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocomplete): Sebuah string. Menentukan salah satu kemungkinan [perilaku autocomplete.](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values)
+* [`autoFocus`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus): Sebuah boolean. Jika `true`, React akan memfokuskan elemen saat pasang.
```
Comment (mazipan) [inline src/content/reference/react-dom/components/input.md:53]: Pasang di sini terjemahan "mount", apa gak dipertahankan aja pakai mount di italic?

**Review by mazipan — APPROVED** [review]

File: src/content/reference/react-dom/components/input.md (hunk 6 above)
Comment (mazipan) [reply src/content/reference/react-dom/components/input.md:53]: Oke

**Review by resir014 — APPROVED** [review]
Waiting on re-review from @r17x

**Review by r17x — APPROVED** [review]

**Pull request merged**